### ISSUE TITLE
VMs not starting, cannot create new ones.

### ISSUE THREAD

**Issue by suotrax** [codesandbox/codesandbox-client#8611]
# 🐛 bug report

## Preflight Checklist

- [X] I have read the [Contributing Guidelines][contributing] for this project.
- [X] I agree to follow the [Code of Conduct][code_of_conduct] that this project
      adheres to.
- [X] I have searched the issue tracker for an issue that matches the one I want
      to file, without success.

## Description of the problem

My VM is restarting and failing. I haven't asked for it. It just started automatically.

"Resuming Micro VM. Still booting"
"MicroVM snapshots are created to boot up in less than 2 seconds"

But nothing happens and secondly, I am getting an error after some time:

Unable to start the microVM
Your changes were saved, but we could not start the microVM running your project. Please try restarting the microVM by clicking the "Restart microVM" button."
All Systems Operational
We didn't encounter any issues in our system, which indicates that the issue might be related to your account.

If the issue persists, please contact our support team at [support@codesandbox.io](mailto:support@codesandbox.io) for further assistance, making sure to copy the error details below, or check the [status page](https://codesandboxstatus.statuspage.io/).

Error: (502) - URL: https://codesandbox.io/api/beta/sandboxes/branches/<<branch_code>>/instance
- DATA: 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
<script>(function(){function c(){var b=a.contentDocument||a.contentWindow.document;if(b){var d=b.createElement('script');d.innerHTML="window.__CF$cv$params={r:'8c1ee4232b3db236',t:'MTcyNjEzMzYxNS4wMDAwMDA='};var a=document.createElement('script');a.nonce='';a.src='/cdn-cgi/challenge-platform/scripts/jsd/main.js';document.getElementsByTagName('head')[0].appendChild(a);";b.getElementsByTagName('head')[0].appendChild(d)}}if(document.body){var a=document.createElement('iframe');a.height=1;a.width=1;a.style.position='absolute';a.style.top=0;a.style.left=0;a.style.border='none';a.style.visibility='hidden';document.body.appendChild(a);if('loading'!==document.readyState)c();else if(window.addEventListener)document.addEventListener('DOMContentLoaded',c);else{var e=document.onreadystatechange||function(){};document.onreadystatechange=function(b){e(b);'loading'!==document.readyState&&(document.onreadystatechange=e,c())}}}})();</script></body></html>
 
- ERROR: SyntaxError: Unexpected token '<', "
<html><hea"... is not valid JSON

Version: 0.0.360
Environment: production
Event: Initial connect to Pitcher
State: ERROR
Branch: <<branch_code>>

## How has this issue affected you? What are you trying to accomplish?

I cannot start any environment. I cannot create a new environment. 

[code_of_conduct]: https://github.com/codesandbox/codesandbox-client/blob/master/CODE_OF_CONDUCT.md
[contributing]: https://github.com/codesandbox/codesandbox-client/blob/master/CONTRIBUTING.md

**Comment by CompuIves** [comment]
Hello! Do you still have this problem? We had a disruption around the time of your message due to a sudden spike in traffic. It should've been fixed relatively quickly after, and we're currently working on adding more nodes to the pool to handle a spike like this in the future.

**Comment by crapaman** [comment]
I cant access my vps for same reason idk why

**Comment by motorcycles-afk** [comment]
> Hello! Do you still have this problem? We had a disruption around the time of your message due to a sudden spike in traffic. It should've been fixed relatively quickly after, and we're currently working on adding more nodes to the pool to handle a spike like this in the future.

I'm probably late but yeah this is happening today too

**Comment by motorcycles-afk** [comment]
it seems Pitcher Manager US-1
is down but im not very sure

**Comment by CompuIves** [comment]
> it seems Pitcher Manager US-1
> is down but im not very sure 

You're right, we had a DoS from within the network (from a VM) around that time. It's resolved now and we've added some measures to ensure that this doesn't happen again. Apologies for the inconvenience!
